CHANGELOG.md

```diff
@@ -1,15 +1,85 @@
+   placed on the closing transaction when we are not the party paying it. To
+   control the feerate paid on a channel at close-time, use
+   `ChannelManager::close_channel_with_target_feerate` instead of
+   `close_channel` (#1011).
+ * `lightning_background_processor::BackgroundProcessor` now stops the
+   background thread when dropped (#1007). It is marked `#[must_use]` so that
+   Rust users will receive a compile-time warning when it is immediately
+   dropped after construction (#1029).
+ * Total potential funds burn on force-close due to dust outputs is now limited
+   to `lightning::util::config::ChannelConfig::max_dust_htlc_exposure_msat` per
+   channel (#1009).
+ * The interval on which `lightning::ln::peer_handler::timer_tick_occurred`
+   should be called has been reduced to once every five seconds (#1035) and
+   `lightning::ln::channelmanager::ChannelManager::timer_tick_occurred` should
+   now be called on startup in addition to once per minute (#985).
+ * The rust-bitcoin and bech32 dependencies have been updated to their
+   respective latest versions (0.27 and 0.8, #1012).
+
+## Bug Fixes
+ * Fix panic when reading invoices generated by some versions of c-lightning
+   (#1002 and #1003).
+ * Fix panic when attempting to validate a signed message of incorrect length
+   (#1010).
+ * Do not ignore the route hints in invoices when the invoice is over 250k
+   sats (#986).
+ * Fees are automatically updated on outbound channels to ensure commitment
+   transactions are always broadcastable (#985).
+ * Fixes a rare case where a `lightning::util::events::Event::SpendableOutputs`
+   event is not generated after a counterparty commitment transaction is
+   confirmed in a reorg when a conflicting local commitment transaction is
+   removed in the same reorg (#1022).
+ * Fixes a remotely-triggerable force-closure of an origin channel after an
+   HTLC was forwarded over a next-hop channel and the next-hop channel was
+   force-closed by our counterparty (#1025).
+ * Fixes a rare force-closure case when sending a payment as a channel fundee
+   when overdrawing our remaining balance. Instead the send will fail (#998).
+ * Fixes a rare force-closure case when a payment was claimed prior to a
+   peer disconnection or restart, and later failed (#977).
 
 ## Serialization Compatibility
+ * Pending inbound keysend payments which have not either been failed or
+   claimed when serialized will result in a `ChannelManager` which is not
+   readable on pre-0.0.100 clients (#967).
+ * If
+   `lightning::chain::keysinterface::KeysInterface::get_shutdown_scriptpubkey`
+   returns a `ShutdownScript` which was constructed using a constructor other
+   than `ShutdownScript::new_p2wpkh_from_pubkey`, the `ChannelManager` will not
+   be readable by versions prior to 0.0.100. Further, versions prior to 0.0.100
+   may not be able to read all `ChannelMonitorUpdate`s generated by 0.0.100
+   (#1019).
```

Comment:
Maybe this should be phrased to describe when this is a problem (i.e. when `commit_upfront_shutdown_pubkey` is not set).